python/ray/_private/utils.py

```diff
@@ -1899,3 +1899,20 @@ def try_import_each_module(module_names_to_import: List[str]) -> None:
             importlib.import_module(module_to_preload)
         except ImportError:
             logger.exception(f'Failed to preload the module "{module_to_preload}"')
+
+
+def update_envs(env_vars: Dict[str, str]):
+    if env_vars is None:
+        return
+    replaceable_keys = [
```

Comment:
Should we support it generically like what Linux does where you can reference any variable via `$var` or `${var}`, not limited to these four.

For example, how to achieve something like `export MY_PATH=${PATH}:~/usr/bin`.


Not sure if there is any use case for that yet and what you did can be an incremental path towards it.

Reply:
There are almost no such scenarios.
1. Because other variables are completely user-defined. There are no such environment variables in the cluster environment. 
2. And other environment variables may have "${}", we replaced it but there is a problem.

Reply:
Ok, sg. We can reconsider this when such use case actually comes up.